src/js/organisms/tabbed-panels.js

```diff
@@ -41,36 +40,36 @@ module.exports = class TabbedPanels {
    * Sets the initial state on elements.
    */
   setInitialState() {
-    this.backgrounds.forEach((background, index) => {
-      if (index) background.classList.remove('displayed');
+    this.contentItems.forEach((background, index) => {
+      if (index) background.classList.remove('is-active');
     });
-    if (this.toggles.length > 1) {
-      this.nav.classList.remove('hidden');
-    }
   }
 
   /**
    * Set CSS classes on elements based on location.hash values.
    */
   showContent() {
     const { hash } = window.location;
+
     if (!hash) return;
 
-    this.toggles.forEach(toggle => {
-      if (toggle.attributes.getNamedItem('href').value === hash) {
-        toggle.classList.add('highlight2');
-        toggle.classList.remove('highlight2-inverse');
+    const hashes = this.contentItems.map(el => `#${el.dataset.hash}`);
+
+    if (!hashes.includes(hash)) return;
```

Comment:
I was going to ask if `includes` is supported in IE11 – it isn’t but we have a polyfill for it 👍